respy/pre_processing/process_covariates.py

```diff
@@ -138,6 +138,51 @@ def separate_covariates_into_core_dense_mixed(options, optim_paras):
     return options
 
 
+def separate_choice_restrictions_into_core_dense_mixed(options, optim_paras):
```

Comment:
Do we need this? If we can create the data for each core index just by catching all exceptions in `create_inadmissible`, then we should not write a function which is complex like this one, right? In Python EAFP (easier to ask for forgiveness than permission) is encouraged.

Reply:
I think this is outdated kept it for reference but should be deleted.